btx/processing/indexer.py

```diff
@@ -51,7 +51,9 @@ def _retrieve_paths(self, taskdir, tag):
         tag : str
             filename extension suffix
         """
-        self.lst = os.path.join(taskdir ,f'r{self.run:04}/r{self.run:04}.lst')
+        if ( tag_cxi != ''):
+            tag_cxi = '_'+tag_cxi
+        self.lst = os.path.join(taskdir ,f'r{self.run:04}/r{self.run:04}{tag_cxi}.lst')
         self.stream = os.path.join(taskdir, f'r{self.run:04}_{tag}.stream')
```

Comment:
Does the stream file nomenclature also need to be updated to _{tag}_{tag_cxi}?

Reply:
Thanks, let's look into that when improving the whole event code business?